sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -554,7 +554,7 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
 
     webpackCliVersion := "2.0.11",
 
-    version in startWebpackDevServer := "2.11.1",
+    version in startWebpackDevServer := "3.1.14", //"2.11.1",
```

Comment:
I think the latest is 3.4.1 perhaps we should directly there, and I'd remove the commented old version